### DISCUSSION TITLE
Öğrenciler ve dersleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#454] [Soru Cevap]
İki kız ve üç erkek öğrencinin kayıtlı olduğu bir okulda, öğrencilerin bu dönem aldıkları dersler ile ilgili aşağıdaki bilgiler verilmiştir.

- Matematik dersine çift sayıda öğrenci kayıtlıdır
- Fizik dersinde kız öğrencilerin sayısı erkek öğrencilere eşittir
- Ali ve Fatma Kimya dersine kayıtlı öğrencilerden ikisidir
- Ahmet ve Mehmet Matematik dersine kayıtlı öğrencilerden ikisidir
- Biyoloji dersini 3 öğrenci almaktadır
- Bir öğrenci bir dönemde tam olarak 3 derse kayıtlıdır
- Bir derste en az bir tane öğrenci kayıtlı olmalıdır
- Ali ve Mehmet hem Biyoloji hem de Türkçe derslerine kayıtlıdır
- Fatma Fizik dersine kayıtlı değildir
- Türkçe dersine hiçbir kız öğrenci kayıtlı değildir

Yukarıda verilen bilgiler ışığında aşağıdaki soruları yanıtlayınız.

**Comment by sonsuzus** [comment]
Kimya dersini kaç öğrenci almaktadır?

**Reply by Eceg25** [reply]
Kimya dersini 3 öğrenci almaktadır: Ayşe, Ali, Fatma.

**Comment by sonsuzus** [comment]
Ayşe hangi dersleri almaktadır?

**Reply by Eceg25** [reply]
Ayşe fizik, matematik ve kimya almaktadır.

**Comment by sonsuzus** [comment]
Fatma ve Ayşe’nin ortak aldığı dersler hangileridir?

**Reply by Eceg25** [reply]
Ortak olarak kimya ve matematik almaktadırlar.

**Comment by sonsuzus** [comment]
Matematik dersini Ahmet ve Mehmet’e ek olarak kimler almaktadır?

**Reply by Eceg25** [reply]
Ahmet ve Mehmet'e ek olarak Ayşe ve Fatma almaktadır.

**Comment by sonsuzus** [comment]
Fizik dersini kimler almaktadır?

**Reply by Eceg25** [reply]
Fizik dersini Ahmet ve Ayşe almaktadır.
